PotreeConverter/src/main.cpp

```diff
@@ -95,7 +99,7 @@ PotreeArguments parseArguments(int argc, char **argv){
 	args.addArgument("edl-enabled", "Enable Eye-Dome-Lighting.");
 	args.addArgument("show-skybox", "");
 	args.addArgument("material", "RGB, ELEVATION, INTENSITY, INTENSITY_GRADIENT, CLASSIFICATION, RETURN_NUMBER, SOURCE, LEVEL_OF_DETAIL");
-
+    args.addArgument("flat_buffer,b", "FlatBuffer file type,  points ||   Lanes");
```

Comment:
```suggestion
    args.addArgument("flatbuffer-type,b", "FlatBuffer file type,  points ||   lanes");
```
Just to fit with the current style for other flags...also we will likely end up having multiple different types, is this a boolean flag right now?